src/tests/ftest/telemetry/telemetry_pool_space_metrics.py

```diff
@@ -0,0 +1,148 @@
+        self.scm_metadata_max_size = self.params.get(
+            "metadata_max_size", "/run/scm_metric_thresholds/*")
+        self.pool_space_metrics_minmax = {
+            "/run/pool_scm/*": {
+                "engine_pool_vos_space_scm_used": (
+                    self.data_size - (self.data_size * self.scm_threshold_percent) / 100,
+                    self.data_size + (self.data_size * self.scm_threshold_percent) / 100
+                ),
+                "engine_pool_vos_space_nvme_used": (0, 0)
+            },
+            "/run/pool_scm_nvme/*": {
+                "engine_pool_vos_space_scm_used": (1, self.scm_metadata_max_size),
+                "engine_pool_vos_space_nvme_used": (self.data_size, self.data_size)
+            }
+        }
+
+    def get_expected_values_range(self, namespace):
+        """Return the expected metrics value output.
+
+        This function returns a hash map of pairs defining min and max values of each tested
+        telemetry metrics.  The hash map of pairs returned depends on the pool created with the
+        given namespace and the size of the data written in it.
+
+        Args:
+            namespace (string): Namespace of the last created pool.
+
+        Returns:
+            expected_values (dict): Dictionary of the expected metrics value output.
+        """
+
+        self.assertIn(
+            namespace, self.pool_space_metrics_minmax,
+            "Invalid pool namespace: {}".format(namespace))
+
+        return self.pool_space_metrics_minmax[namespace]
+
+    def get_metrics(self, names):
+        """Obtain the specified metrics information.
+
+        Args:
+            name (list): List of metric names to query.
+
+        Returns:
+            metrics (dict): a dictionary of metric keys linked to their aggregated values.
+        """
+        metrics = {}
+        for name in names:
+            metrics[name] = 0
+
+        for data in self.telemetry.get_metrics(",".join(names)).values():
+            for name, value in data.items():
+                for metric in value["metrics"]:
+                    metrics[name] += metric["value"]
+
+        return metrics
+
+    def test_telemetry_pool_space_metrics(self):
+        """JIRA ID: DAOS-10192
+
+        Create a file of 16MiB thanks to ior to verify the DAOS engine IO telemetry vos space
+        metrics.
+
+        Steps:
+            Create a pool
+            Create a container
+            Generate deterministic workload.  Using ior to write 128MiB of data.
+            Use telemetry command to get value of vos space metrics "engine_pool_vos_space_scm_used"
+            and "engine_pool_vos_space_nvme_used" for all targets.
+            Verify the sum of all parameter metrics matches the workload.
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=telemetry
+        :avocado: tags=telemetry_pool_space_metrics,test_telemetry_pool_space_metrics
```

Comment:
We've started using the class name as the tag (preferred)
```suggestion
        :avocado: tags=TelemetryPoolSpaceMetrics,test_telemetry_pool_space_metrics
```
Or, if you want both (less common)
```suggestion
        :avocado: tags=TelemetryPoolSpaceMetrics,telemetry_pool_space_metrics,test_telemetry_pool_space_metrics
```

Reply:
Fixed with commit 77f5e57